style.css

```diff
@@ -232,3 +233,77 @@ li {
   justify-content: center;
   align-items: center;
 }
+
+/* Footer section */
+.footer-container {
+  margin-top: 2em;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+
+  /* prettier-ignore */
+  background: url(./assest/illustration-contact-form.svg) no-repeat 100% 90%;
+  text-align: center;
+  padding: 2em 5em;
+}
+
+form {
+  margin-top: 2em;
+}
+
+footer a {
+  text-decoration: none;
+  color: #42526e;
+  font-size: 20px;
+}
+
+.form-fields {
+  list-style: none;
+  display: flex;
+  width: max-content;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 0;
+}
+
+input,
+textarea {
+  width: 327px;
+  padding: 1em;
+  margin: 1em 0;
+  border-radius: 5px;
+}
+
+footer > a {
+  margin-top: 1.5em;
+}
+
+input:valid,
+textarea:valid {
+  border: 2px solid #008552;
+}
+
+input:invalid,
+textarea:invalid {
+  border: 2px solid red;
+}
+
+input:invalid:required,
+textarea:invalid:required {
```

Comment:
- [ ] [REQUIRED] Please ensure you stick to the design templates in figma as much as possible, this includes the font, colours, images and spacing (margin/padding). You can change the background color in the input form to match the original
## YOUR DESIGN
![Capture11](https://user-images.githubusercontent.com/63964646/144120032-f21672eb-5a5f-4d72-a379-2bec212cf964.PNG)
## ORIGINAL DESIGN
![Capture12](https://user-images.githubusercontent.com/63964646/144120059-298fca38-1734-495e-bd35-d2bcfec8bcba.PNG)

Reply:
Thanks, @lily-coder,

I thought I could adjust some to my liking, thanks for the information, now I will strictly abide by the design